src/types/registration.types.ts

```diff
@@ -79,9 +79,11 @@ interface RegistrationPublisher {
   id: string;
 }
 
-type ImportSourceName = 'Brage' | 'Cristin' | 'Scopus';
+type AdditionalIdentifierType = 'CristinIdentifier' | 'ScopusIdentifier';
+type ImportSourceName = 'Cristin' | 'Scopus';
```

Comment:
Hvorfor tok du bort Brage her?

Reply:
Fordi den uansett ikke brukes, og jeg ikke fant noen poster som hadde en verdi for det heller (selv om det bare var noen stikkprøver da)

Reply:
ImportDetails bruker Brage